client/state/data-layer/wpcom-http/test/utils.js

```diff
@@ -9,10 +9,10 @@ import {
 	getProgress,
 	dispatchRequest,
 	dispatchRequestEx,
-	makeParser,
 	reducer,
 	trackRequests,
 } from '../utils.js';
+import { SchemaError } from 'lib/make-json-schema-parser';
```

Comment:
Can we replace `SchemaError` with just `Error` and remove the dependency on `lib/make-json-schema-parser` completely?